tests/gpu/linalg_test/Directsum_test.cpp

```diff
@@ -24,11 +24,11 @@ namespace DirectsumTest {
     axes:{1}
   ====================*/
   TEST(Directsum, gpu_allDType) {
-    for (auto device : device_list) {  // now only test for cpu device.
+    for (auto device : device_list) {
       for (auto dtype1 : dtype_list) {
         for (auto dtype2 : dtype_list) {
-          Tensor T1 = Tensor({12, 5, 7}, dtype1, device);
-          Tensor T2 = Tensor({12, 5, 8}, dtype2, device);
+          Tensor T1 = Tensor({12, 5, 7}, dtype1, device).to(cytnx::Device.cuda);
```

Comment:
The device_list in test_tools.h seems wrong. Could you create an issue to address it?

Reply:
I have leave the issue #548 .